doc/user_doc/vic_admin/upgrade_vch_options.md

```diff
@@ -2,6 +2,8 @@
 
 The command line utility for vSphere Integrated Containers Engine, `vic-machine`, provides an `upgrade` command that allows you to upgrade virtual container hosts to a newer version. The options that `vic-machine upgrade` requires depend on the location in your vSphere environment in which you deployed the virtual container host to upgrade.
 
+**IMPORTANT**: Due to the substantial changes in vSphere Integrated Containers version 0.7, you cannot use `vic-machine upgrade` to upgrade from version 0.6 to version 0.7. You can use `vic-machine upgrade` to upgrade from more recent builds to version 0.7.
```

Comment:
"to upgrade from more recent builds to version 0.7", we might only guarantee upgrade from GA to newer version. From 0.7 to GA will be best to have.
So not sure if it's okay to declare this. 😄